configs/centerpoint/nusc_centerpoint_pp_02voxel_circle_nms.py

```diff
@@ -241,7 +241,7 @@
 # yapf:enable
 # runtime settings
 total_epochs = 20
-device_ids = range(8)
+device_ids = range(1)
```

Comment:
Why do we need to run this on just 1 gpu? (is it for training?) I think we can use overcap to get multiple GPUs, at once, right?

Reply:
This is just for local testing (as I've got just one GPU on my computer). I'll use another config file for training on skynet.